Profiler_Test/ProfilerTest.cs

```diff
@@ -75,13 +75,8 @@ public int TestProcessSelection(string process)
         public int TestConfigFile(string regex)
         {
             var configFile = Path.Combine(TestTempDirectory, "profilerconfig.yml");
-            File.WriteAllText(configFile, $@"
-match:
-  - profiler:
-      enabled: false
-  - executablePathRegex: {regex}
-    profiler:
-      enabled: true
+            File.WriteAllText(configFile, $@" 
+match:       [{{profiler: {{enabled         : false}}}}, {{executablePathRegex: {regex}     , profiler: {{enabled: true}}}}]

```

Comment:
Is the point of this to test different formatting? If so I'd just create a second test for that purpose and leave the existing test as is.